### DISCUSSION TITLE
Question(s)

### DISCUSSION THREAD

**Discussion by Fireworkstars46** [Dhranios/Luigi-s-Mansion#103] [Q&A]
As for the ghosts in the map,  is the trick to suck them fast up mode is sneak and walk backwards in rounds back and forth or hold sneak and walk backwards or?

**Comment by Dhranios** [comment]
No sneaking, just switching between moving away from the ghost and not repeatedly. Don't worry about walls, there's a system in place that allows you to keep doing that against a wall too.

This is mentioned in the FAQ BTW.

**Reply by Fireworkstars46** [reply]
Also after I save it from the toad, how exactly do I load the save?

**Reply by Dhranios** [reply]
Q. Can I disregard current data somehow?
A. Of course, just talk to E. Gadd to load your old data.

Additionally you can also do this after dying.

**Reply by Fireworkstars46** [reply]
Also would the map work in multiplayer? Like lan single player but open to lan thing. And also can I no enter the breaker room after the fight with Chauncey and the 1F hallway is first unlocked?

**Reply by Dhranios** [reply]
Q. Is the map multiplayer compatible?
A. Absolutely! The map even provides different translation strings based on player count so languages can properly pluralize them.
There is, however, an issue with rapid teleportation causing desyncs, see next question.

I suggest reading the FAQ :)

The breaker room door is only locked during the blackout.

**Reply by Fireworkstars46** [reply]
Cause I just unlocked the 1F hallway after the fight with Chauncey and went right to the breaker room and can't enter it some reason (was unlocked but can't walk in it).

**Reply by Dhranios** [reply]
v3.0 will release this month, which has rewritten a lot, and is thuroughly being tested.
This problem won't persist in that version.

**Reply by Fireworkstars46** [reply]
I'm on the 1.18.2 version of the map

**Reply by Dhranios** [reply]
That version is 2.5 years older than latest release (which is also almost a year old), may things changed and got fixed since.

**Reply by Fireworkstars46** [reply]
So would the bug get fixed in it or?
